man/bundle-install.ronn

```diff
@@ -228,26 +111,29 @@ gems into their final location.
 ## INSTALLING GROUPS
 
 By default, `bundle install` will install all gems in all groups
-in your Gemfile(5), except those declared for a different platform.
+in your Gemfile(5), except those declared for a different platform
+than the platform your installing your gems into.
 
 However, you can explicitly tell Bundler to skip installing
-certain groups with the `--without` option. This option takes
+certain groups with the `without` config. This config takes
 a space-separated list of groups.
 
-While the `--without` option will skip _installing_ the gems in the
+     $ bundle config set without test development
+
+While the `without` config will skip _installing_ the gems in the
 specified groups, it will still _download_ those gems and use them to
 resolve the dependencies of every gem in your Gemfile(5).
 
 This is so that installing a different set of groups on another
  machine (such as a production server) will not change the
 gems and versions that you have already developed and tested against.
 
-`Bundler offers a rock-solid guarantee that the third-party
+Bundler offers a rock-solid guarantee that the third-party
```

Comment:
> +Bundler offers a rock-solid guarantee that the third-party

Not a big deal, but I'd remove "rock-solid". The guarantee should be enough :) (and some folks might see that and think _eye roll_)